src/main/resources/static/PasswordValidation.js

```diff
@@ -0,0 +1,27 @@
+function validatePassword(){
```

Comment:
file name - to make it easier to distinguish between java and js please write the file name small
-> passwordValidation.js